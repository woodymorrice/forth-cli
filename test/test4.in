."hello" true 3.1415 ."greetings" false 23760
