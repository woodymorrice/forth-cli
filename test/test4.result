1c1
< bad3
---
> good
