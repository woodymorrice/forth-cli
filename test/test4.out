hello
true hello
3.1415 true hello
greetings 3.1415 true hello
false greetings 3.1415 true hello
23760.0000 false greetings 3.1415 true hello
